css/style.css

```diff
@@ -0,0 +1,553 @@
+    height: 188px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 120px;
+    height: 188px;
+  }
+
+  .level:hover::before {
+    display: none;
+  }
+}
+
+@media (max-width: 768px) {
+  .table-0 {
+    width: 580px;
+  }
+
+  .table-1 {
+    width: 580px;
+    height: 600px;
+  }
+
+  .card-wrapper {
+    width: 180px;
+    height: 282px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 180px;
+    height: 282px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 620px) {
+
+  .menu {
+    max-width: 92%;
+    text-align: center;
+  }
+
+  .first-card,
+  .second-card {
+    display: none;
+  }
+
+  .level {
+    margin-left: 0;
+  }
+
+  .lvl-active::after {
+    right: 77%;
+  }
+
+  .play {
+    display: block;
+    margin: 40px auto;
+  }
+
+  .level-choice__header {
+    font-size: 48px;
+  }
+
+  .level-choice__instruction {
+    margin-bottom: 100px;
+    font-size: 20px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 155px;
+    height: 242px;
+  }
+
+  .card-wrapper {
+    width: 155px;
+    height: 242px;
+  }
+
+  .table-0 {
+    width: 500px;
+  }
+
+  .table-1 {
+    width: 500px;
+    height: 520px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 540px) {
+
+  .level {
+    margin-left: 0;
+  }
+
+  .lvl-active::after {
+    right: 82%;
+  }
+
+  .play {
+    min-width: 250px;
+  }
+
+  .level-choice__header {
+    font-size: 40px;
+  }
+
+  .level-choice__instruction {
+    margin-bottom: 70px;
+  }
+
+  .front-side,
+  .back-side {
+    width: 120px;
+    height: 188px;
+  }
+
+  .card-wrapper {
+    width: 120px;
+    height: 188px;
+  }
+
+  .table-0 {
+    width: 400px;
+  }
+  
+  .table-1 {
+    width: 400px;
+    height: 420px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 430px) {
+
+  .lvl-active::after {
+    right: 88%;
+  }
+
+  .front-side,
+  .back-side {
+    width: 100px;
+    height: 156px;
+  }
+
+  .card-wrapper {
+    width: 100px;
+    height: 156px;
+  }
+
+  .table-0 {
+    width: 340px;
+  }
+
+  .table-1 {
+    width: 340px;
+    height: 360px;
+  }
+
+  .table-2 {
+    width: 175px;
+    height: 675px;
+  }
+
+  .table-2 .card-wrapper {
+    width: 80px;
+    height: 125px;
+  }
+
+  .table-2 .front-side,
+  .table-2 .back-side {
+    width: 80px;
+    height: 125px;
+  }
+}
+
+@media (max-width: 375px) {
+
+  .lvl-active::after {
+    right: 93%;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)